### ISSUE TITLE
The download action button on a users proposal list intermittently fails

### ISSUE THREAD

**Issue by ACLay** [UserOfficeProject/issue-tracker#718]
## Which role does this affect.

This was reported by someone logged in as a User. They only seemed to get failures on their proposal list page. Selecting the "DOWNLOAD PDF" button on an individual proposal's page seemed to work fine.

## Steps to reproduce (if it's a bug).

Click the download action button on the users homepage.

## Other helpful details like screenshots or email content from.

This screenshot was included with the report. The failure happened for multiple of these proposals, but non consistently.

![My_proposals_1 (002)](https://user-images.githubusercontent.com/2938284/194281856-581ab67b-e12c-47f9-86eb-16883e631f01.png)

**Comment by simonfernandes** [comment]
This morning I couldn't download any proposals at all - regardless of whether they were my own or which round they were from. Using an incognito window fixed it for me.

**Comment by simonfernandes** [comment]
@mutambaraf Do you think this may potentially have been solved by your recent PDF work?

**Comment by mutambaraf** [comment]
This might require monitoring but it looks related.

**Comment by simonfernandes** [comment]
**Notes from meeting with Sukh, Simon, Philip and Emma 12th Dec 2022:**

There might be two issues here:

1. Downloading a PDF fails with an error message and requires the user to use an incognito browser.
2. Even in incognito (which Emma exclusively uses), sometimes the download button just continually spins. Clicking cancel and retrying usually then works.

Emma doesn't see this as too much of an issue currently as you can usually get things to work with a bit of fiddling.

**Comment by georgeflecknell** [comment]
An instrument scientist, Stephen King just complained about this problem. He has the issue with the spinning wheel and then having to reload the page to be able to download. This is quite frequently reported as a problem and Stephen is calling it a major barrier to his work. Please could we look at fixing it?

**Comment by mutambaraf** [comment]
@georgeflecknell if it is possible will be very grateful to get more information.

- What browser they are using.
- The webpage they are trying to download from.
- RB numbers of the proposal(s).

**Comment by georgeflecknell** [comment]
User: Instrument Scientist Svemir Rudic
Site: https://proposal.facilities.rl.ac.uk/
Proposals affected: 2310016
More from the user:
I have now tried to download all the proposals on my list, 033562, 2310017, 2310016, 2310014, and I was successful in all cases.
Actually, when I tried to download 2310016, initially it did not work, but after trying 033562 and succeeding, I tried 2310016 again, and was then successful.
It is good to see that things are working, although occasionally one may experience difficulties (for no obvious reason).
Browser used: Desktop Chrome
